person.rb

```diff
@@ -1,17 +1,18 @@
 require_relative 'nameable'
 
 class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id, :rentals
```

Comment:
- [ ] Here again `rental` should be marked with `attr_accessor`, as we intend to both _access (get)_ as well as _write (set)_ it. Kindly ensure to improve upon this aspect.

Reply:
```ruby
class Person < Nameable
  attr_accessor :name, :age, :rentals

``` 
Resolved issue.